raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -454,6 +456,8 @@ contract TokenNetwork is Utils {
     )
         public
     {
+        // Convention required in getSettleTransferAmounts
+        require(participant2_transferred_amount >= participant1_transferred_amount);
```

Comment:
I think you could interchange `participant1` and `participant2` here appropriately if needed, so we have the assumption, but transactions do not fail.